[Dashboard] Add configurability of 'orgId' param for requesting Grafana dashboards (#53236)
ray-project/ray: doc/source/cluster/configure-manage-dashboard.md

Configure these settings using the `RAY_GRAFANA_HOST`, `RAY_PROMETHEUS_HOST`, `RAY_PROMETHEUS_NAME`, and `RAY_GRAFANA_IFRAME_HOST` environment variables when you start the Ray Clusters.

* Set `RAY_GRAFANA_HOST` to an address that the head node can use to access Grafana. Head node does health checks on Grafana on the backend.

> * Set `RAY_GRAFANA_ORG_ID` to the organization id which you will used in Grafana. Default is "1".

Reviewer: ```suggestion
* Set `RAY_GRAFANA_ORG_ID` to set the organization id to be used in Grafana. Default is "1".
```

> * Set `RAY_GRAFANA_ORG_ID` to the organization id which you will used in Grafana. Default is "1".

Reviewer 2: ```suggestion
* Set `RAY_GRAFANA_ORG_ID` to the organization ID you use in Grafana. Default is "1".
```
